xml/System/TimeOnly.xml

```diff
@@ -902,10 +929,11 @@
         <Parameter Name="formats" Type="System.String[]" />
       </Parameters>
       <Docs>
-        <param name="s">To be added.</param>
-        <param name="formats">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <param name="s">A span containing the characters that represent a time to convert.</param>
+        <param name="formats">An array of allowable formats of s.</param>
+        <summary>Converts the specified span representation of a time to its TimeOnly equivalent using the specified array of formats.
+            The format of the string representation must match at least one of the specified formats exactly or an exception is thrown.</summary>
```

Comment:
@tarekgh this `ParseExact` method, as well as all the overloads below, mention that an exception is thrown, but do not document it. Can you please add them under the `<returns></returns>` element?

Reply:
I added the exception info to all parsing methods in DateOnly and TimeOnly.